util/util.go

```diff
@@ -9,6 +9,10 @@ import (
 	"strings"
 )
 
+const (
+	OSRELEASE_PATH = "/etc/os-release"
```

Comment:
Think of this as something that will grow to include all the supervisor functionality. So we need to have separate packages for shared functionality that  will be moved to separate repos eventually. `util` and `resin` are separated because they are used both by the CLI (client) and provisioner (server).
I think it does make sense to have a `resin` package, as it's the one with the interactions with the resin API.